tools/build/Makefile.vars

```diff
@@ -82,10 +82,15 @@ LIB_ASAN_PATH =
 PREFIX := $(patsubst "%",%,$(PREFIX))
 EXTRA_BINS :=
 
+SOL_VERSION := 0
+SOL_CURRENT := 0
+SOL_REVISION := 0
+SOL_AGE := 0
+
 export PKGNAME := soletta
-export VERSION_MAJOR := 0
-export VERSION_MINOR := 0
-export VERSION_RELEASE := 1
+export VERSION_MAJOR := $(shell echo $$(($(SOL_CURRENT) - $(SOL_AGE))))
```

Comment:
do we have a way to execute this math without doing shell operation? We already use lots of GNU make features, so using http://gmsl.sourceforge.net/ could be simpler:

```
export VERSION_MAJOR := $(call subtract,$(SOL_CURRENT),$(SOL_AGE))
```